drivers/dac/ad5791/ad5791.c

```diff
@@ -408,11 +408,67 @@ int ad5791_set_lin_comp(struct ad5791_dev *dev,
 		return -EINVAL;
 	}
 
-	if (!v_span)
+	if (v_span)
```

Comment:
this fix could be in a different commit, with the according fixes tag

Reply:
Please add the fixes tag, above the Signed-off-by tag in the commit message as follows:
```
Fixes: b527970 ("drivers: dac: ad5791: Add linearity error compensation API")
```